ci/tasks/build.yml

```
@@ -0,0 +1,30 @@
+
+outputs:
+  - name: stemcell
+  - name: stemcells-index
+    path: stemcells-index-output
+  - name: candidate-build-number
+
+run:
+  path: bosh-stemcells-ci/ci/tasks/build.sh
+
+params:
+  IAAS:            replace-me
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)